core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/requirements.txt

```diff
@@ -145,6 +144,7 @@ kubernetes-cni-0.7.5-0
 kubernetes-cni-0.8.6-0
 
 [files]
+https://github.com/rabbitmq/erlang-rpm/releases/download/v23.1.5/erlang-23.1.5-1.el7.x86_64.rpm
```

Comment:
Is there an arm64 alternative for this package?

Reply:
so, for arm i will suggest to use the packages directly from erlang-solution platform, they support amd and arm arch. The packages from erlang-solution are a little bit bigger (all in one).